index.html

```diff
@@ -49,5 +52,170 @@ <h3>I am a Software Developer</h3>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+	</section>
+
+	<section class="about-me">
+		<div class="container2">
+			<h1>About Me</h1>
+			<p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and 
+			   experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			   <button class="btn">Get My Resume</button>
+			   <img src="images/about img.png" alt="Vector shapes">
+		</div>
+
+		<div class="tools">
+			<h3><img src="images/diamond.png" alt="">Language</h3>
+			<ul>
+				<li>Javascript</li>
+				<li>Ruby</li>
+				<li>Html</li>
+				<li>Css</li>
+			</ul>
+		</div>
+
+		<div class="tools">
+			<h3><img src="images/Rectangle.png" alt="">Frameworks</h3>
+			<ul>
+				<li>Bootstrap</li>
+				<li>Ruby on rail</li>
+				<li>RSPec</li>
+				<li>Copy Bara</li>
+				<li>Selenium</li>
+			</ul>
+		</div>
+
+		<div class="tools">
+			<h3><img src="images/circle.png" alt="">Skills</h3>
+			<ul>
+				<li>Codekit</li>
+				<li>Github</li>
+				<li>Codepen</li>
+				<li>Gitlab</li>
+				<li>Terminal</li>
+			</ul>
+		</div>
+	</section>
+
+	<section class="message">
+		<div class="container2">
+			<h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+			<div class="input">
+				<p>Full Name</p>
+				<p>Email Address</p>
+				<div class="msg">
+				<p class="msg1">Enter text here</p>
+			</div>
+			</div>
+			<button class="btn">Get in touch</button>
+		</div>
+	</section>
+
+	<footer class="icons">
+		<div class="contact">
+			<ul class="social">
+			<li><a href=""><i class="fa fa-github"></i></a></li>
+			<li><a href=""><i class="fa fa-linkedin"></i></a></li>
+			<li><a href=""><i><img src="images/Vector.png" alt="esty logo"></i></a></li>
+			<li><a href=""><i class="fa fa-twitter"></i></a></li>
+			<li><a href=""><i class="fa fa-g-mail"><h3>M</h3></i></a></li>
+		</ul>
+		</div>
+	</footer>
```

Comment:
- [ ] Great job for creating a form but This is not a requirement for this project kindly remove this code. :slightly_smiling_face: